service/handler/tx.go

```diff
@@ -34,7 +34,10 @@ func HandleTx(block *types.Block) error {
 			}
 			batch = append(batch, txOp)
 		}
+		//TODO: return txOps and save it
 		handleProposal(tx)
+		txOps := handleTokenFlow(blockWithTags, tx)
```

Comment:
Pass the array pointer into the handleTokenFlow function, adding array elements by function, which eliminates unnecessary intermediate variables